vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -28,15 +33,24 @@
     "pattern-sd-with-small-logo-bar",
 ]
 
-project_dir = Path(__file__).parents[3]
-hash_folder = project_dir / SAMPLE_HASH_FOLDER
-video_folder = project_dir / SAMPLE_VIDEOS
```

Comment:
(ignorable): Ugh, I'm starting to remember this mess now

Reply:
If it makes you feel better it gets even worse if you look at `setup.py` and `vpqd-release.py` for vpdq.

Reply:
I think we ran into a time/polish tradeoff with the original author. We're running a small staff. You are currently talking at the only maintainer, and I also have the ThreatExchange API and our hash import/export pipelines on my duties at Meta. Would love to spend more time on our open source (and am blocking some time out in preparation for a hackathon in October).

Hopefully hiring goes well and we'll be able to spend more time. Other than that, I'm incredibly grateful for folks like you who can spend some time.